lib/services/resource.go

```diff
@@ -218,6 +218,8 @@ func ParseShortcut(in string) (string, error) {
 		return types.KindAuditQuery, nil
 	case types.KindSecurityReport:
 		return types.KindSecurityReport, nil
+	case types.KindServerInfo, types.KindServerInfo + "s":
```

Comment:
The word "information" does not have a plural form in English, so nor does the abbreviation "info". The `+ "s"` case should be removed.